pysat/tests/test_utils_io.py

```diff
@@ -214,29 +239,116 @@ def test_inst_write_and_read_netcdf(self):
                                 var, attr, repr(ival),
                                 repr(netcdf_inst.meta[var, attr])))
 
-        del netcdf_inst
+        del netcdf_inst.data, netcdf_inst
+
+        # Delete the file. This doesn't work on Windows. It does work if
+        # executed before the netCDF is loaded. Despite our .close() statement
+        # in loading code, there are still open references.
+        # os.remove(outfile)
+
```

Comment:
If we're removing this, let's cut all the code not comment it out.

Reply:
I updated the comment with a TODO referencing the xarray issue. I left that code there since it is the result of time spent debugging to sort out where the error is. It is useful for the future in fully resolving #974.